pkg/agent/datamodel/types.go

```diff
@@ -1054,9 +1055,11 @@ func (a *AgentPoolProfile) GetKubernetesLabels(rg string, deprecated bool) strin
 		buf.WriteString(",kubernetes.io/role=agent")
 	}
 	buf.WriteString(fmt.Sprintf(",agentpool=%s", a.Name))
+	storagetier, _ := common.GetStorageAccountType(a.VMSize)
 	if strings.EqualFold(a.StorageProfile, ManagedDisks) {
-		storagetier, _ := common.GetStorageAccountType(a.VMSize)
 		buf.WriteString(fmt.Sprintf(",storageprofile=managed,storagetier=%s", storagetier))
+	} else if strings.EqualFold(a.StorageProfile, Ephemeral) {
+		buf.WriteString(fmt.Sprintf(",storageprofile=ephemeral,storagetier=%s", storagetier))
```

Comment:
hold off till confirm:

Storage tier will always be `Standard_LRS` for ephemeral.